code/game/objects/items/devices/radio/headset.dm

```diff
@@ -274,7 +274,7 @@
 
 /obj/item/device/radio/headset/almayer/ce
 	name = "chief engineer's headset"
-	desc = "The headset of the guy who is in charge of morons. To access the engineering channel, use :e. For command, use :v."
+	desc = "The headset of the guy who is in charge of morons. To access the logistics channel, use :u. For command, use :v."
```

Comment:
Is `:l` taken?

Reply:
Currerntly piggybacking off of `SUP_FREQ` which is tied to U. 

Can switch over to :l after I wade into that.